res_syntax/src/res_core.ml

```diff
@@ -2009,20 +2008,18 @@ and parseBracketAccess p expr startPos =
     let accessExpr = parseConstrainedOrCoercedExpr p in
     Parser.expect Rbracket p;
     Parser.eatBreadcrumb p;
-    let rbracket = p.prevEndPos in
-    let arrayLoc = mkLoc lbracket rbracket in
     match p.token with
     | Equal ->
       Parser.leaveBreadcrumb p ExprArrayMutation;
       Parser.next p;
       let rhsExpr = parseExpr p in
       let arraySet =
-        Location.mkloc (Longident.Ldot (Lident "Array", "set")) arrayLoc
+        Location.mknoloc (Longident.Ldot (Lident "Array", "set"))
       in
       let endPos = p.prevEndPos in
       let arraySet =
         Ast_helper.Exp.apply ~loc:(mkLoc startPos endPos)
```

Comment:
For now, this could be alternatives:
```ocaml
Ast_helper.Exp.apply ~loc:(mkLoc lbracket rbracket)
```
What do you think?

Reply:
The example case's ouput looks like this:
```
// This comment will vanish

(
  incidents
  ->Belt.Array.keep(({status}) => status === #OPEN)
  ->Belt.SortArray.stableSortBy((a, b) => compare(a.createdTime, b.createdTime))
)[0]
```
The comment location looks weird, but at least we wouldn't lost the loc for error msg.

Reply:
I have a separate question. Why does it matter what location one gives to array functions given that they are not printed? I'd like to see where in the printing mechanism that matters, and whether one can make it be ignored.